spec/directions/bake_chapter_introductions_spec.rb

```diff
@@ -179,4 +234,120 @@
     end
   end
 
+  context 'when v2 called on book without chapter outline' do
+    it 'works' do
+      described_class.v2(book: book_with_diff_order, chapter_objectives_strategy: :default)
+      expect(book_with_diff_order.body).to match_normalized_html(
+        <<~HTML
+          <body>
+            <div data-type="chapter">
+              <h1 data-type="document-title">Chapter 1 Title</h1>
+              <div class="introduction" data-type="page">
+                <figure class="splash">
+                  <div data-type="title">Blood Pressure</div>
+                  <figcaption>A proficiency in anatomy and physiology... (credit: Bryan Mason/flickr)</figcaption>
+                  <span data-type="media" data-alt="This photo shows a nurse taking a woman&#x2019;s...">
+                  <img src="ccc4ed14-6c87-408b-9934-7a0d279d853a/100_Blood_Pressure.jpg" data-media-type="image/jpg" alt="This photo shows a nurse taking..."/>
+                  </span>
+                </figure>
+                <div class="intro-body">
+                  <div data-type="note" class="chapter-objectives">
+                    <div data-type="title">Chapter Objectives</div>
+                    <p>After studying this chapter, you will be able to:</p>
+                    <ul>
+                      <li>Distinguish between anatomy and physiology, and identify several branches of each</li>
+                    </ul>
+                  </div>
+                  <div class="intro-text">
+                    <h2 data-type="document-title"><span data-type="" itemprop="" class="os-text">Introduction</span></h2>
+                    <p id="123">Though you may approach a course in anatomy and physiology...</p>
+                    <p id="123_copy_1">This chapter begins with an overview of anatomy and...</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </body>
+        HTML
+      )
+    end
+  end
```

Comment:
@KarinaMendez2 In which book we can find this case? I think it is a mistake in old recipes. All notes should be modified like in case from below, so content should be wrapped into `os-note-body` and title should be modified to h3. I guess that someone didn't add note with class `chapter-objectives` to `Config_Note`. It should be done like that in cnx-recipes:

![image](https://user-images.githubusercontent.com/40228854/129334703-020fbd97-1edc-449d-bfc6-b17b6bbbe460.png)

Reply:
This case of not having chapter-outline is Anatomy & P, first case I run into